test/tests/val-2f/tests

```diff
@@ -0,0 +1,46 @@
+[Tests]
+  design = 'EquilibriumBC.md'
+  issues = '#12'
+  validation = 'val-2f.md'
+  [val-2f_heavy_csv]
+    type = CSVDiff
+    input = val-2f.i
+    cli_args = 'Outputs/file_base=val-2f_heavy_out'
+    csvdiff = val-2f_heavy_out.csv
+    heavy = true
+    requirement = 'The system shall be able to model diffusion of deuterium in a beryllium sample and generate CSV data output.'
+  []
+  [val-2f_heavy_exodus]
+    type = Exodiff
+    input = val-2f.i
+    cli_args = 'Outputs/file_base=val-2f_heavy_out'
+    exodiff = val-2f_heavy_out.e
+    prereq = val-2f_heavy_csv
+    should_execute = false # this test relies on the output files from val-2f_heavy_csv, so it shouldn't be run twice
+    heavy = true
+    abs_zero = 1e-8
```

Comment:
```suggestion
```
Let's remove this until we realize that you need it later on. This should not be used by default.